encode/av1encode.c

```diff
@@ -25,6 +25,12 @@
         exit(1);                                                        \
     }
 
+#define CHECK_FN(cod, func)                                                     \
```

Comment:
s/cod/cond/ (+ fix same typo in CHECK_CONDITION down below)

Reply:
replace it with CHECK_CONDITION(cond, func)